frontend/core-ui/src/modules/automations/components/builder/nodes/components/WorkflowCanvasSheet.tsx

```
@@ -0,0 +1,251 @@
+
+    if (workflowIndex < 0) {
+      return;
+    }
+
+    onMembersChanged?.();
+    outerForm.setValue(
+      `workflows.${workflowIndex}.actions` as FieldPath<TAutomationBuilderForm>,
+      innerActions,
+      { shouldDirty: true },
+    );
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)